packages-common/pluggable-widgets-generator/generators/app/index.js

```diff
@@ -0,0 +1,284 @@
+                `${this.widget.templateSourcePath}${widgetSrcFolder}BadgeSample.${jsxFileExtension}.ejs`,
+                `${widgetSrcFolder}BadgeSample.${jsxFileExtension}`
+            );
+            this._copyTemplate(
+                `${this.widget.templateSourcePath}src/WidgetName.${jsxFileExtension}.ejs`,
+                `src/${widgetName}.${jsxFileExtension}`
+            );
+        }
+
+        if (this.widget.isPlatformWeb) {
+            this._copyTemplate(
+                `${this.widget.templateSourcePath}src/WidgetName.editorPreview.${jsxFileExtension}.ejs`,
+                `src/${widgetName}.editorPreview.${jsxFileExtension}`
+            );
+            this._copyTemplate(`${this.widget.templateSourcePath}src/ui/WidgetName.css`, `src/ui/${widgetName}.css`);
+
+            if (this.widget.usesFullTemplate) {
+                this._copyFile(
+                    `${this.widget.templateSourcePath}${widgetSrcFolder}Alert.${jsxFileExtension}.ejs`,
+                    `${widgetSrcFolder}Alert.${jsxFileExtension}`
+                );
+            }
+        } else {
+            const fileExtension = this.widget.isLanguageTS ? "ts" : "js";
+
+            if (this.widget.usesFullTemplate) {
+                this._copyFile(
+                    `${this.widget.templateSourcePath}src/ui/styles.${fileExtension}`,
+                    `src/ui/styles.${fileExtension}`
+                );
+            }
+            this._copyFile(
+                `${this.widget.templateSourcePath}src/utils/common.${fileExtension}`,
+                `src/utils/common.${fileExtension}`
+            );
+        }
+    }
+
+    _writeCompilerOptions() {
+        if (this.widget.isLanguageTS) {
+            this._copyFile("commons/tsconfig.json", "tsconfig.json");
+        }
+    }
+
+    _writeUtilityFiles() {
+        this._copyFile("commons/_gitignore", ".gitignore");
+        this._copyFile(`commons/eslintrc.${this.widget.isLanguageTS ? "ts" : "js"}.js`, ".eslintrc.js");
+        this._copyFile("commons/prettier.config.js", "prettier.config.js");
+        this._copyFile("commons/.gitattributes", ".gitattributes");
+
+        if (this.widget.license) {
+            switch (this.widget.license.toLowerCase()) {
+                case "apache-2.0":
+                    this._copyTemplate("licenses/APACHE2", "LICENSE");
+                    break;
+                case "mit":
+                    this._copyTemplate("licenses/MIT", "LICENSE");
+                    break;
+                default:
+                    break;
+            }
+        }
+    }
+
+    _writeUnitTests() {
+        const extension = this.widget.isLanguageTS ? "tsx" : "jsx";
+
+        if (this.widget.hasUnitTests) {
+            if (this.widget.isPlatformWeb) {
+                if (this.widget.usesFullTemplate) {
+                    this._copyFile(
+                        `${this.widget.templateSourcePath}${widgetSrcFolder}/__tests__/Alert.spec.${extension}.ejs`,
```

Comment:
We use `ejs` extension here but no special extension for other templates like README

Reply:
Use `ejs` everywhere where templating occurs.